### PR TITLE
chore: fix git allow-protocol for all jest

### PR DESCRIPTION
**Pull request by rarkins**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "generate": "run-s generate:*",
     "generate:imports": "node tools/generate-imports.mjs",
     "git-check": "node tools/check-git-version.mjs",
-    "jest": "cross-env LOG_LEVEL=fatal jest --logHeapUsage",
+    "jest": "cross-env LOG_LEVEL=fatal GIT_ALLOW_PROTOCOL=file jest --logHeapUsage",
```
Comment (viceice) [inline package.json:25]: where did you found that variable?

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
we could also set that variable in git tests only

**Comment by rarkins** [comment]
> we could also set that variable in git tests only

I tried just setting process.env in the test which fails but it didn't work. Maybe `simple-git` loads env at startup?

File: package.json (hunk 1 above)
Comment (rarkins) [reply package.json:25]: I forget exactly where

Comment (HonkingGoose) [reply package.json:25]: @viceice and @rarkins I think this is the documentation for the `GIT_ALLOW_PROTOCOL` variable:

- https://git-scm.com/docs/git#Documentation/git.txt-codeGITALLOWPROTOCOLcode
- https://git-scm.com/docs/git-config#Documentation/git-config.txt-protocolallow

**Pull request merged**